### PR TITLE
Create a libhv example project

### PR DESCRIPTION
**Pull request by leamus**
libhv example project

### CODE REVIEW

File: cpp/libhv/main.cpp [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        res->body = req->query_params.find("id")->second;
+        return 200;
+    });
+    service.POST("user", [](HttpRequest* req, HttpResponse* res) {
+        res->body = "";
+        return 200;
+    });
+
+    http_server_t server;
+    server.port = 8888;
```
Comment (waghanza) [inline cpp/libhv/main.cpp:22]:
```suggestion
    server.port = 3000;
```

**Review by waghanza — CHANGES_REQUESTED** [review]

**Comment by waghanza** [comment]
is the server available from `0.0.0.0` ?

**Comment by leamus** [comment]
yes

**Comment by waghanza** [comment]
could you create all `cmake` manifest please ?

you can take inspiration from `drogon` in **cpp** folder

**Comment by leamus** [comment]
OK~

**Comment by waghanza** [comment]
@leamus do you wish to continue this PR ?

**Pull request closed without merging**